src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/README.md

```diff
@@ -0,0 +1,56 @@
+```
+const Table = require('./Table').default;
+const Header = require('./Header').default;
+const Body = require('./Body').default;
+const Row = require('./Row').default;
+const Cell = require('./Cell').default;
+const HeaderCell = require('./HeaderCell').default;
+
+const tableData = {
+    header: ['Type', 'Name', 'Author', 'Date', 'Subversion', 'Uploadgröße', 'Dateigröße'],
+    body: [
+        ['Blog', 'Meine ersten 100 Tage MASSIVE ART', 'Adrian Sieber', '24.12.2017', 'Github', '20 MB', 'Test'],
+        ['Blog', 'Meine ersten 100 Tage MASSIVE ART', 'Adrian Sieber', '24.12.2017', 'Github', '20 MB', 'Test'],
+        ['Blog', 'Meine ersten 100 Tage MASSIVE ART', 'Adrian Sieber', '24.12.2017', 'Github', '20 MB', 'Test'],
+        ['Blog', 'Meine ersten 100 Tage MASSIVE ART', 'Adrian Sieber', '24.12.2017', 'Github', '20 MB', 'Test'],
+        ['Blog', 'Meine ersten 100 Tage MASSIVE ART', 'Adrian Sieber', '24.12.2017', 'Github', '20 MB', 'Test'],
+        ['Blog', 'Meine ersten 100 Tage MASSIVE ART', 'Adrian Sieber', '24.12.2017', 'Github', '20 MB', 'Test'],
+        ['Blog', 'Meine ersten 100 Tage MASSIVE ART', 'Adrian Sieber', '24.12.2017', 'Github', '20 MB', 'Test'],
+    ],
+};
+
+<Table>
+    <Header>
+        <Row>
+            {
+                tableData.header.map((headerCell, index) => {
```

Comment:
Please just write a simple table without arrays and map calls here, so that it is as easy to understand the basics of the component as possible.